README.md

```diff
@@ -19,6 +19,51 @@ https://alchemycodelab.github.io/web-01-character-designer/
 
 User [this repository](https://github.com/alchemycodelab/half-baked-web-01-character-designer) as a template for this deliverable.
 
+
+MAKE A PLAN:
+- Identify any missing HTML elements/ids/classes and fill them in
+- Confirm DOM elements correspond to correct HTML elements with appropriate ids/classes
+- Create missing DOM elements (if there are any)
+- Follow comments in JS chronologically
+    // set state for how many times the user changes the head, middle, and bottom
+    // set state for all of the character's catchphrases
+
+
+     // get the value of the head dropdown
+    // increment the head change count state
+    // update the dom for the head (use style.backgroundImage on the bottomEl div instead of trying to set the .src -- it's NOT an img tag!)
+    // update the stats to show the new count (call displayStats() to do this work)
+
+
+    // get the value of the middle dropdown
+    // increment the middle change count state
+    // update the dom for the middle (NOTE: use style.backgroundImage on the middleEl div instead of trying to set the .src -- it's NOT an img tag!)
+    // update the stats to show the new count (call displayStats() to do this work)
+
+
+     // get the value of the bottom dropdown
+    // increment the bottom change count state
+    // update the dom for the bottom (NOTE use style.backgroundImage on the bottomEl div instead of trying to set the .src -- it's NOT an img tag!)
+    // update the stats to show the new count (call displayStats() to do this work)
+
+
+    // get the value of the catchphrase input
+    // push the new catchphrase to the catchphrase array in state
+    // clear out the form input's value so it's empty to the user
+    // update the dom to show the new catchphrases (refactor to/call displayCatchphrases to do this work)
+
+
+    // text content of the reportEl to tell the user how many times they've changed each piece of the state
+
+
+    // clear out the DOM for the currently displayed catchphrases
+    // loop through each catchphrase in state
+    // and for each catchphrase
+    // create an HTML element with the catchphrase as its text content
+    // and append that HTML element to the cleared-out DOM
+
```

Comment:
yes planning! this makes me happy every time. you're going to gain such a thorough understanding of your code by doing this.